x-pack/plugins/security_solution/cypress/e2e/exceptions/alerts_table_flow/endpoint_exceptions.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { createRule } from '../../../tasks/api_calls/rules';
+import {
+  waitForAlertsToPopulate,
+  waitForTheRuleToBeExecuted,
+} from '../../../tasks/create_new_rule';
+import {
+  esArchiverLoad,
+  esArchiverResetKibana,
+  esArchiverUnload,
+} from '../../../tasks/es_archiver';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import {
+  addExceptionFlyoutItemName,
+  selectCloseSingleAlerts,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import { ALERTS_COUNT, EMPTY_ALERT_TABLE } from '../../../screens/alerts';
+import {
+  EXCEPTION_ITEM_CONTAINER,
+  FIELD_INPUT_PARENT,
+  NO_EXCEPTIONS_EXIST_PROMPT,
+} from '../../../screens/exceptions';
+import {
+  removeException,
+  goToAlertsTab,
+  goToEndpointExceptionsTab,
+} from '../../../tasks/rule_details';
+
+describe('Endpoint Exceptions workflows from Alert', () => {
+  const expectedNumberOfAlerts = 1;
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('endpoint');
+    login();
+    deleteAlertsAndRules();
+    createRule(getEndpointRule());
+  });
+  beforeEach(() => {
+    visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+    goToRuleDetails();
+    waitForTheRuleToBeExecuted();
+    waitForAlertsToPopulate();
+  });
+  after(() => {
+    esArchiverUnload('endpoint');
+  });
+
+  it('Should be able to create and close single Endpoint exception from overflow menu', () => {
+    // The Endpoint will populated with predefined fields
+    openAddEndpointExceptionFromFirstAlert();
+
+    // As the endpoint.alerts-* is used to trigger the alert the
+    // file.Ext.code_signature will be populated as the first item
+    cy.get(EXCEPTION_ITEM_CONTAINER)
+      .eq(0)
+      .find(FIELD_INPUT_PARENT)
+      .eq(0)
+      .should('have.text', 'file.Ext.code_signature');
+
+    // The Endpoint will populated with predefined fields
+    openAddEndpointExceptionFromFirstAlert();
+
+    selectCloseSingleAlerts();
+    addExceptionFlyoutItemName('Sample Exception');
+    submitNewExceptionItem();
+
+    // Alerts table should now be empty from having added exception and closed
+    // matching alert
+    cy.get(EMPTY_ALERT_TABLE).should('exist');
+
+    // Closed alert should appear in table
+    goToClosedAlertsOnRuleDetailsPage();
+    cy.get(ALERTS_COUNT).should('exist');
+    cy.get(ALERTS_COUNT).should('have.text', `${expectedNumberOfAlerts} alert`);
+
+    // Endpoint Exception will move to Endpoint List under Exception tab of rule
+    goToEndpointExceptionsTab();
+
+    // Remove the exception and load an event that would have matched that exception
+    // to show that said exception now starts to show up again
+    removeException();
+    // when removing exception and again, no more exist, empty screen shows again
+    cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('exist');
+
+    // load more docs
```

Comment:
Careful with the below lines: even though we are adding new docs, we are still on the `closed alerts` tab, so we are not asserting that new alerts have been generated, the one displayed is still the old one.

Moreover we are waiting for the rule to be executed to perform the assertion, but, the created rule is executed every `100m` so it has not been really executed. 

We should improve this piece of code by:
- Enabling and disabling the rule to force the execution
- Navigate to the `open alerts` tab and make sure that a new alert has been generated.

Reply:
Thank you Glo! please check [0887b7b](https://github.com/elastic/kibana/pull/152757/commits/0887b7b36b356f1db6810fe12d97133de8837bd7)